x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/rules_stats_collector.ts

```diff
@@ -65,141 +73,153 @@ const getRulesStatsQuery = (): SearchRequest => ({
   query: {
     match_all: {},
   },
+  runtime_mappings: getIdentifierRuntimeMapping(),
   aggs: {
-    rules: {
+    accounts: {
       terms: {
-        field: 'rule.id',
+        field: 'asset_identifier',
         order: {
           _count: 'desc',
         },
         size: 100,
       },
       aggs: {
-        rule_name: {
-          top_metrics: {
-            metrics: {
-              field: 'rule.name',
-            },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
-            },
-          },
-        },
-        rule_section: {
-          top_metrics: {
-            metrics: {
-              field: 'rule.section',
-            },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
-            },
-          },
-        },
-        rule_version: {
-          top_metrics: {
-            metrics: {
-              field: 'rule.version',
-            },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
-            },
-          },
-        },
-        posture_type: {
-          top_metrics: {
-            metrics: {
-              field: 'rule.benchmark.posture_type',
-            },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
+        rules: {
+          terms: {
+            field: 'rule.id',
+            order: {
+              _count: 'desc',
             },
+            size: 100,
```

Comment:
IMO this should be a constant that equals to the amount of rules we currently have. or just a really big number that we will never meet. 100 sounds small enough that we can miss some rules in the future when we have more than 100.